vue/components/form/inputs/base.ts

```diff
@@ -46,13 +42,14 @@ export default {
     },
   },
   methods: {
-    isValidNumber(num) {
-      const notValidValues = [undefined, Number.NaN, ''];
+    isValidNumber(num: any) {
+      const notValidValues: any[] = [undefined, Number.NaN];
```

Comment:
po mojem ne bi bila slaba rešitev, če bi Field dobil metodo validateValue(value), ki bi rešila zgornji TODO in izločila kodo ven iz base.ts. Hkrati bi dejansko podprli client-side validatorje in za začetek implementirali integervalidator? Dejva prediskutirat.